src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/introduction-opentelemetry-new-relic.mdx

```diff
@@ -159,19 +159,19 @@ APM agents offer detailed transaction trace visibility for individual services.
 
 ## How OpenTelemetry works with New Relic [#how-it-works]
 
-Here are two basic ways you could set up OpenTelemetry with New Relic: use the OpenTelemetry collector or use a New Relic exporter. Each approach assumes you set up instrumentation in your code, but they differ in how the data is gathered and sent to New Relic.
+Here are two basic ways you could set up OpenTelemetry with New Relic: use the OpenTelemetry collector (recommended), or use the native OTLP exporter (pre-release). Each approach assumes you set up instrumentation in your code, but they differ in how the data is gathered and sent to New Relic.
```

Comment:
The OpenTelemetry project's OTLP exporters for each language are in various states of development, and at least Java and Python's are GA. Perhaps you mean that it's our New Relic *OTLP endpoint* that's in pre-release?

Reply:
Yes, good point @rvanderwal-newrelic, I'llfollow-up with a fix PR to replace _OTLP exporter_ with _OTLP endpoint_.

Reply:
https://github.com/newrelic/docs-website/pull/1975